terraform/variables.tf

```diff
@@ -10,7 +10,7 @@ variable "azs" {
 
 variable "public_url" {
   type    = string
-  default = "echo.walletconnect.com"
+  default = "https://echo.walletconnect.com"
```

Comment:
@Rakowskiii I think its the `https://` prefix here causing some of the terraform issues